examples/oft-solana/programs/endpoint-mock/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
Why is the endpoint-mock bundled with the example code?  Wouldn't it make sense to make it a separate package so as to not bloat the example code?

Reply:
You need the RegisterOApp method I believe, which is why it's included here.

Reply:
Yes, this is used in unit testing, and I haven’t found a better way to handle this issue yet.

Reply:
At this point, we have no process to publish crates but yeah it would be nice to have it packaged.

Possible solution is to move it to `packages` directory and have cargo install it using `git`, I'll investigate